src/pages/Subscribe/Subscribe.js

```diff
@@ -1,7 +1,362 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { useNavigate, useLocation } from 'react-router-dom';
 
 const Subscribe = () => {
-  return <h1>Subscribe</h1>;
+  const { pathname } = useLocation();
+
+  const [index, setIndex] = useState(1);
+  const [name, setName] = useState('');
+  const [address, setAddress] = useState('');
```

Comment:
```suggestion  
  const[userInfo, setUserInfo] = useState({
  index:1,
  name:"",
  address:""
  })
```

* 비슷한 유형의 정보를 담는다면, 하나씩 따로따로 할수도 있지만, 
* 한번에 객체 혹은 배열로 관리할 수 있습니다.